dev/doxygen/pages/translations.md

```diff
@@ -0,0 +1,27 @@
+Translation Guidelines {#doc_translations}
+==========================================
+
+[TOC]
+
+# Key Terms {#doc_translations_terms}
+
+For consistent translations, certain LibrePCB specific terms (like "package" or
+"device") should always be translated in the same way.
+
+| **EN** | DE |
+|--------|----|
+| **Pin** | Pin |
+| **Symbol** | Symbol |
+| **Symbol Variant** | Symbol-Variante |
+| **Component** | Komponente |
+| **Device** | Bauteil |
+| **Part** | Typ |
+| **Package** | Gehäuse |
+| **Pad** | Pad |
+| **Footprint** | Footprint |
+| **Workspace** | Workspace |
+| **Project** | Projekt |
+| **Library** | Library |
+| **Schematic** | Schema |
+| **Board** | Board |
```

Comment:
We could translate this as "Platine", but I like "Board" more.

Reply:
Yeah I think I would also go with "Board".